pkg/flowaggregator/flowaggregator.go

```diff
@@ -458,6 +483,74 @@ func (fa *flowAggregator) flowRecordExpiryCheck(stopCh <-chan struct{}) {
 			klog.V(4).InfoS("Total number of records exported", "count", fa.numRecordsExported)
 			klog.V(4).InfoS("Total number of flows stored in Flow Aggregator", "count", fa.aggregationProcess.GetNumFlows())
 			klog.V(4).InfoS("Number of exporters connected with Flow Aggregator", "count", fa.collectingProcess.GetNumConnToCollector())
+		case msg := <-fa.updateCh:
+			switch msg.param {
+			case updateExternalFlowCollectorAddr:
+				//modify addr and proto if changes.
+				newAddr := msg.value.(querier.ExternalFlowCollectorAddr)
+				if newAddr.Address == fa.externalFlowCollectorAddr && newAddr.Protocol == fa.externalFlowCollectorProto {
+					continue
+				}
+				klog.InfoS("Updating flow-collector address")
+				fa.externalFlowCollectorAddr = newAddr.Address
+				fa.externalFlowCollectorProto = newAddr.Protocol
+				klog.InfoS("Config ExternalFlowCollectorAddr is changed", "address", fa.externalFlowCollectorAddr, "protocol", fa.externalFlowCollectorProto)
+				if fa.exportingProcess != nil {
+					fa.exportingProcess.CloseConnToCollector()
+					fa.exportingProcess = nil
+				}
+			case enableClickHouse:
+				klog.Info("Enabling ClickHouse")
+				chInput := msg.value.(clickhouseclient.ClickHouseInput)
+				err := fa.InitDBExportProcess(chInput)
+				if err != nil {
+					klog.ErrorS(err, "Error when creating db export process")
+					continue
+				}
+				klog.InfoS("Clickhouse param is:", "database", chInput.Database, "databaseURL", chInput.DatabaseURL, "debug", chInput.Debug, "compress", *chInput.Compress, "commitInterval", fa.dbExportProcess.GetCommitInterval().String(), "ttl", fa.dbExportProcess.GetTTL())
+				go fa.dbExportProcess.Start()
+				defer fa.dbExportProcess.Stop()
+			case updateClickHouseParam:
+				chInput := msg.value.(clickhouseclient.ClickHouseInput)
+				dsn, connect, err := clickhouseclient.CheckConnection(chInput)
+				if err != nil {
+					klog.ErrorS(err, "Error when checking new connection:")
```

Comment:
you don't need to add a colon at the end of such log messages

Reply:
I've fixed it. Thanks.